CHANGELOG/CHANGELOG-1.9.md

```diff
@@ -0,0 +1,74 @@
+# Changelog 1.9
+
+## 1.9.0 - 2022-10-21
+
+### Added
+
+- Add the following capabilities to the Multi-cluster feature:
+  * Add support for Pod to Pod connectivity across clusters. ([#4219](https://github.com/antrea-io/antrea/pull/4219), [@hjiajing])
+  * Add active-passive mode high availability support for Gateway Nodes. ([#4069](https://github.com/antrea-io/antrea/pull/4069), [@luolanzone])
+  * Allow Pod IPs as Endpoints of Multi-cluster Service; option `endpointIPType` is added to Multi-cluster Controller ConfigMap to specify the Service Endpoints type. ([#4198](https://github.com/antrea-io/antrea/pull/4198), [@luolanzone])
+  * Add `antctl mc get joinconfig` command to print ClusterSet join parameters. ([#4299](https://github.com/antrea-io/antrea/pull/4299), [@jianjuns])
+  * Add `antctl mc get|delete membertoken` commands to get/delete member token. ([#4254](https://github.com/antrea-io/antrea/pull/4254), [@bangqipropel])
+- Add rule name to Audit Logging for Antrea-native policies. ([#4178](https://github.com/antrea-io/antrea/pull/4178), [@qiyueyao])
+- Add Service health check similar to kube-proxy in antrea-agent; it provides HTTP endpoints "<nodeIP>:<healthCheckNodePort>/healthz" for querying number of local Endpoints of a Service. ([#4120](https://github.com/antrea-io/antrea/pull/4120), [@shettyg])
+- Add S3Uploader as a new exporter of Flow Aggregator, which periodically exports expired flow records to AWS S3 storage bucket. ([#4143](https://github.com/antrea-io/antrea/pull/4143), [@heanlan])
+- Add scripts and binaries needed for running Antrea on non-Kubernetes Nodes (ExternalNode) in release assets. ([#4266](https://github.com/antrea-io/antrea/pull/4266) [#4113](https://github.com/antrea-io/antrea/pull/4113), [@antoninbas] [@Anandkumar26])
+
+### Changed
```

Comment:
Not related to this specific PR, I am wondering what's the rule to add these items in each section? by the merged sequence?

Reply:
you mean section selection or order of tiems in each section? I guess the latter.
The major of items are in the order when the script "prepare-changelog.sh" collects them, but sometimes we adjusted some items according to their correlation and importance.